packages/react-components/src/components/MessageThread.tsx

```diff
@@ -601,7 +601,7 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
     disableJumpToNewMessageButton = false,
     showMessageDate = false,
     showMessageStatus = false,
-    numberOfChatMessagesToReload = 0,
+    numberOfChatMessagesToReload = 5,
```

Comment:
how is numberOfChatMessagesToReload related to maxPageSize?

Reply:
This one if for rendering and maxPageSize is for network request, this one is more perf related parameter

Reply:
Why was this 0 before?